airbyte-integrations/connectors/source-zendesk-sunshine/source_zendesk_sunshine/manifest.yaml

```diff
@@ -0,0 +1,519 @@
+                      type: CompositeErrorHandler
+                      error_handlers:
+                        - type: DefaultErrorHandler
+                          backoff_strategies:
+                            - type: WaitTimeFromHeader
+                              header: Retry-After
+                    request_body_json: {}
+                  record_selector:
+                    type: RecordSelector
+                    extractor:
+                      type: DpathExtractor
+                      field_path:
+                        - data
+                  paginator:
+                    type: DefaultPaginator
+                    page_token_option:
+                      type: RequestPath
+                    page_size_option:
+                      type: RequestOption
+                      field_name: per_page
+                      inject_into: request_parameter
+                    pagination_strategy:
+                      type: CursorPagination
+                      page_size: 100
+                      cursor_value: '{{ response.get("links", {}).get("next", {}) }}'
+                      stop_condition: '{{ not response.get("links", {}).get("next", {}) }}'
+  - type: DeclarativeStream
+    name: relationship_types
+    primary_key:
+      - key
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://{{ config['subdomain'] }}.zendesk.com/api/sunshine/
+        path: relationships/types
+        http_method: GET
+        request_parameters: {}
+        request_headers:
+          Content-Type: application/json
+        authenticator:
+          type: BasicHttpAuthenticator
+          password: '{{ config[''password''] }}'
+          username: '{{ config[''username''] }}'
+        error_handler:
+          type: CompositeErrorHandler
+          error_handlers:
+            - type: DefaultErrorHandler
+              backoff_strategies:
+                - type: WaitTimeFromHeader
+                  header: Retry-After
+        request_body_json: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - data
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestPath
+        page_size_option:
+          type: RequestOption
+          field_name: per_page
+          inject_into: request_parameter
+        pagination_strategy:
+          type: CursorPagination
+          page_size: 100
+          cursor_value: '{{ response.get("links", {}).get("next", {}) }}'
+          stop_condition: '{{ not response.get("links", {}).get("next", {}) }}'
+
+spec:
+  documentation_url: https://example.org
+  type: Spec
+  connection_specification:
+    $schema: http://json-schema.org/draft-07/schema#
+    type: object
+    additionalProperties: true
+    required:
+      - start_date
+      - subdomain
+    properties:
```

Comment:
The authentication methods (`OAuth2.0` and `API Token`) from the current version's spec.json are not properly configured here. Those authentication values need to be migrated here. You can check [the docs](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/reference) for information on how to configure that.